src/apis/modTools.es6.js

```diff
@@ -1,19 +1,34 @@
 import apiRequest from '../apiBase/apiRequest';
 
+const DISTINGUISH_TYPE = {
+  'moderator': 'yes',
+  '': 'no',
+  'admin': 'admin',
+};
+
 const remove = (apiOptions, fullname, spam) => {
-    // Remove a link, comment, or modmail message.
-    const body = {
-    	id: fullname,
-    	spam: spam
-    };
+  // Remove a link, comment, or modmail message.
+  const body = {
+    id: fullname,
+    spam: spam
+  };
 
-    return apiRequest(apiOptions, 'POST', 'api/remove', { body, type: 'form' });
+  return apiRequest(apiOptions, 'POST', 'api/remove', { body, type: 'form' });
 }
 
 const approve = (apiOptions, fullname) => {
-    // Approve a link or comment
-    const body = { id: fullname };
-    return apiRequest(apiOptions, 'POST', 'api/approve', { body, type: 'form' });
+  // Approve a link or comment
+  const body = { id: fullname };
+  return apiRequest(apiOptions, 'POST', 'api/approve', { body, type: 'form' });
+}
+
+const distinguish = (apiOptions, fullname, distinguishType) => {
+  // Distinguish a link or comment
+  const body = {
+    how: DISTINGUISH_TYPE[distinguishType],
```

Comment:
Should we check if the passed `distinguishType` is a valid value? Maybe we can just default to `null` and just let the api return an error?